agreements/employment.md

```diff
@@ -0,0 +1,167 @@
+    4. Notice to quit will only be effective if given in writing, which includes by e-mail.
+    5. Once notice has been given (whether from you to us, or from us to you), we reserve the right, at any time after the notice was given while you are still employed with us, to give you pay in lieu of notice. This means that we will pay you, as a lump sum, the salary that would otherwise be payable to you during the unexpired portion of your notice period.
+
+9. ### Holiday entitlement
+    1. We operate a holiday year which runs from 1st July to 30th June in the following year.
+    2. Leave accrues throughout the year according to the formula given below, resulting in 33 days paid leave for full time staff and a pro rata proportion for those working part time.
+    3. Unless notified otherwise you will be required to take all Bank Holidays and Public Holidays as leave and those days are counted in the total of 33 days.
+    4. You may carry over a maximum of five days' leave from one holiday year to the next.
+    5. At any time in a leave year, your accrued leave entitlement may be calculated as follows: 2.08 x Number of days worked in the year to date / 21.75
+    6. The amount of leave you may take will be the total leave accrued, rounded up to the nearest half a day, minus any leave you have already taken in the leave year.
+    7. We will usually permit you to take up to five days leave in advance of it being accrued.
+    8. If you wish to take leave, you must obtain approval (which will not be unreasonably withheld) and give notice of:
+        1. 28 days for a period of leave longer than one day;
+        2. seven days for a period of one day or less.
+    9. When your employment ends, you will receive payment in lieu for any part of your leave entitlement which has accrued but which has not been taken. Similarly, if you have taken more leave than you have accrued, we will deduct the appropriate amount of pay from any payment due from us to you on the termination of your employment or, if this is insufficient, we will require you to make up any shortfall.
+
+10. ### Sickness absence
+    1. dxw will pay your normal salary for up to ten days of sickness per year.
+    2. If you are frequently absent as a result of being sick, or are absent for a long period ("persistent sickness absence"), we may have no choice but to dismiss you.
+    3. dxw will not dismiss you for persistent sickness absence without making every reasonable effort to accommodate your circumstances. In particular, we will consider:
+        1. relevant medical information
+        2. options for alternative working arrangements or duties
+        3. your views and preferences
+    4. Further details of dxw's sickness absence arrangements are published in the Playbook.
+
+11. ### Parental leave
+    1. Details of dxw's parental leave arrangements are published in the Playbook.
```

Comment:
The contract contains the basic structure, plus some things that have to be there because of laws (like more holiday detail). The Playbook contains the implementation. Pensions is a good example. The contract says you are entitled to one, the playbook says what it is and how it works.

The contract will be varied infrequently and with a lot of process because if we do change it we are probably changing something quite fundamental about everyone's employment.

The playbook will be varied frequently and with notification rather than advance notice, because they are operational details which should not make fundamental changes to working arrangements.

Playbook guiding principle: it explains how we run the company.